src/mds/mdstypes.cc

```diff
@@ -492,9 +492,9 @@ void feature_bitset_t::dump(Formatter *f) const {
 void feature_bitset_t::print(ostream& out) const
 {
   std::ios_base::fmtflags f(out.flags());
-  out << "0x";
+  out << "0x0";
```

Comment:
What will happen when `_vec[0] = 0x1234567812345678` ? Won't it print `4 + 64` bits ?

Reply:
I don't think `0x1234567812345678` forms a valid feature bit. Do you see any chance?

Reply:
No, I didn't see this. I just mean when the most significant bit is set it will print an extra `0` before it ?

Reply:
I've updated the code considering that.